### PR TITLE
Add Lemmy link to menu

### PR DESCRIPTION
**Pull request by makeasnek**
Add a lemmy link to the menu due to Reddit exodus

### CODE REVIEW

**Comment by nathanielcwm** [comment]
The community was previously not viewable on lemmy.world but that seems to be remedied now.

File: _includes/_header.htm [hunk 1]
```diff
@@ -22,6 +22,8 @@ <h6 class="dropdown-header text-center">Gridcoin Community</h6>
                         <a class="dropdown-item" href="https://www.reddit.com/r/gridcoin">Reddit</a>
                         <a class="dropdown-item" href="https://steemit.com/trending/gridcoin">Steemit</a>
                         <a class="dropdown-item" href="https://twitter.com/GridcoinNetwork">Twitter</a>
+                        <a class="dropdown-item" href="https://lemmy.ml/c/gridcoin">Lemmy</a>
+
```
Comment (barton2526) [inline _includes/_header.htm:26]: why is a new line being added here?

Comment (makeasnek) [reply _includes/_header.htm]: Good point, fixed

**Comment by RoboticMind** [comment]
Should probably be added to the footer as well

**Comment by makeasnek** [comment]
Added

**Pull request merged**